storybookjs/storybook
# Addon(info) - wrap prop values in braces

Issue:
Based on the code around `braceWrap ` I believe the intention was to surround non-string prop values in curly braces. This would make it pretty easy to copy/paste react code from storybook into an app.

## What I did
I surrounded non-string prop values in html encoded curly braces.

## How to test
addon-info "Story Source" used to look like:
```
<Component prop=value />
```
now looks like
```
<Component prop={value} />
```

Is this testable with jest or storyshots?
Yes, with storyshots. I've updated them.

Does this need a new example in the kitchen sink apps?
No.

Does this need an update to the documentation?
No.

If your answer is yes to any of these, please make sure to include it in your PR.
